### DISCUSSION TITLE
New data doesn´t display in elements

### DISCUSSION THREAD

**Discussion by cgrillo98** [CollectionBuilder/collectionbuilder.github.io#116] [Q&A] [answered]
Hello. I need to display new metadata in my collection. I already added the data in my csv, added a new camp in the data section in theme.yml , added the element in config-metadata.csv, but the metadata still doesn´t display in the itms of my collection. Does anyone know what I´m missing?

**Comment by evanwill** [comment]
@cgrillo98 hmm, it sounds like you have things correct. Could you point to the repository so we could take a look? 
The most often cause of this problem is that there is some other error, and GitHub is not actually rebuilding your site. You will see a red X near your commits where there is an error building github pages (a green check next to successful builds). You can look back at the the history to see where the error first started, which should give you an idea of how to fix it!

**Reply by cgrillo98** [reply]
There is a green check. 
The new category is "autor"
This is the link : https://github.com/cdigitaljaveriana/museogif

**Reply by evanwill** [reply]
@cgrillo98 wow, fun collection! 
I am not sure I exactly understand the problem, but here are some things I noticed:

For the "autor" field, the reason it is not displaying as expected, is that in your metadata file [demo-metadata.csv](https://github.com/cdigitaljaveriana/museogif/blob/main/_data/demo-metadata.csv) the header row has an extra space after "autor". It currently looks like this: `creator,autor ,date,`. If you delete that space, like `creator,autor,date,`., then "autor" will start showing up as expected where you configured it. 

Also looking at your collection metadata [demo-metadata.csv](https://github.com/cdigitaljaveriana/museogif/blob/main/_data/demo-metadata.csv), a few of the values in "filename" column do not *exactly* match the filenames of the GIFs in your "objects" folder. When they do not exactly match, the images will not display on your site. For example, `Labofetada.gif` in your metadata, doesn't match any file in your "objects". Another example, your metadata has `esquina_de_la_memoria.gif` but the file in "objects" has spaces instead of underscore like "esquina de la memoria.gif".
Also, you have one filename, `GIFT-CLTDGT#1 (1).gif`, which will not work on the web. The filenames can't have characters like `#` in them. 
For all the filenames, it is best to make them all no spaces, no special characters, then ensure that the metadata exactly matches including the case.

Hope that helps!

**Reply by cgrillo98** [reply, marked as answer]
Thank you very much!!! the extra space was the problem
